### PR TITLE
Cpff

### PR DESCRIPTION
**Pull request by pittlerf**
Hi, I have now successfully compiled.
Cheers
Feri

### CODE REVIEW

**Comment by kostrzewa** [comment]
What's the status on this?

**Comment by kostrzewa** [comment]
Ping

File: contract_cvc_tensor.cpp [hunk 1]
```diff
@@ -1142,20 +1142,20 @@ int contract_write_to_h5_file (double ** const c_tp, void * file, char*tag, cons
         hid_t grp;
         hid_t loc_id = ( grp_list_nmem == 0 ) ? file_id : grp_list[grp_list_nmem-1];
         if ( g_verbose > 1 ) fprintf ( stdout, "# [contract_write_to_h5_file] grp_ptr = %s\n", grp_ptr );
-  
-        grp = H5Gopen2( loc_id, grp_ptr, gapl_id );
-        if ( grp < 0 ) {
-          fprintf ( stderr, "[contract_write_to_h5_file] Error from H5Gopen2 for group %s, status was %ld %s %d\n", grp_ptr, grp, __FILE__, __LINE__ );
+        std::string fail_path;
+        if( check_key_exists(loc_id, grp_ptr, fail_path, false) ){
+          grp = H5Gopen2(loc_id, grp_ptr, gapl_id);
```
[the remark this answers is not in the document]
Comment (kostrzewa) [reply contract_cvc_tensor.cpp]: I repeat: Is this the only occurrence?

**Comment by kostrzewa** [comment]
Also, is there anything new in ```loop_extract``` which should be merged?

**Comment by pittlerf** [comment]
I create a pull request.

**Comment by pittlerf** [comment]
In this pull request I actually made the contractions in contract_loop compatible with your style of hdf5 tags. 
Loop_extract I did not changed.

**Comment by kostrzewa** [comment]
> In this pull request I actually made the contractions in contract_loop compatible with your style of hdf5 tags.

and you've caught all occurrences of ```H5Gopen2``` called on non-existent groups?

>  Loop_extract I did not changed.

I know that you did not change it here, I'm asking in general if there have been any changes that should be tracked.

**Comment by pittlerf** [comment]
> > In this pull request I actually made the contractions in contract_loop compatible with your style of hdf5 tags.
> 
> and you've caught all occurrences of `H5Gopen2` called on non-existent groups?
> 
> > Loop_extract I did not changed.
> 
> I know that you did not change it here, I'm asking in general if there have been any changes that should be tracked.

No, I have not changed that. I left the format as it was previously. I have looked for all occurrences of H5Gopen2.

**Pull request merged**